magento/graphql-ce
# GraphQl-220: Implement exception logging

### Description (*)
Created logging for client and server errors.
I use exception category from \GraphQL\Error\ClientAware to divide errors between the client (`var/log/graphql/client/exception.log`) and server (`var/log/graphql/server/exception.log`).
In case the category is not recognized, the errors will be logged into a general log (`var/log/graphql/exception.log`)


### Fixed Issues (if relevant)
1. magento/graphql-ce#220: Implement exception logging


### Manual testing scenarios (*)
1. Cause any client-side error (Incorrect query parameter, for example) and check `var/log/graphql/client/exception.log`
2. Manually throw exception elsewhere on backend during query processing, and check `var/log/graphql/server/exception.log`

### Contribution checklist (*)
